zulipterminal/ui_tools/boxes.py

```diff
@@ -582,7 +593,8 @@ def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
                     self.view.controller.save_draft_confirmation_popup(
                         this_draft,
                     )
-        elif is_command_key('CYCLE_COMPOSE_FOCUS', key):
+        elif (is_command_key('CYCLE_COMPOSE_FOCUS', key)
+              or is_primary_direction_key(key)):
```

Comment:
Why are capturing all the direction keys with the method? Wouldn't `GO_RIGHT` and `GO_DOWN` suffice?

Reply:
Actually, yes they should.
I may have misinterpreted [one of Neil's point from above](https://github.com/zulip/zulip-terminal/pull/864#pullrequestreview-591884868), and thought we needed to handle cycling for the "up" key as well. I'll amend the commit to use only `GO_RIGHT` and `GO_DOWN`. Then, I guess we won't be needing the second commit at all? I'll remove that too and keep a backup of this branch just in case. :)